### ISSUE TITLE
Open YouTube link on Whatsapp desktop

### ISSUE THREAD

**Issue by YJethno** [uBlockOrigin/uBlock-issues#2680]
### Prerequisites

- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] using a new, unmodified browser profile.

### Description

When you click a YouTube link on WhatsApp Desktop, the link opens in your browser, but ads are not blocked.

This won't happen if the browser is already running. This happens when the browser isn't running, and when you open a link it launches anew.

### A specific URL where the issue occurs.

https://youtu.be/5vj42kKsTBI

### Steps to Reproduce

1. Open WhatsApp Desktop
2. Copy & Paste any YouTube link
3. Make sure any browsers are not running
4. Click the YouTube link

### Expected behavior

1. Ads are blocked
2. Contents play

### Actual behavior

1. Ads play in advance

### Configuration

Configuration

**Comment by iam-py-test** [comment]
Are you using Chrome or a Chromium-based browser (i.e. Brave)?
If so, then there is not anything uBo can do

**Comment by YJethno** [comment]
> Are you using Chrome or a Chromium-based browser (i.e. Brave)? If so, then there is not anything uBo can do

I am currently using Microsoft Edge, yes, a Chromium-based.

**Comment by YJethno** [comment]
> Enable _Suspend network activity until all filter lists are loaded_

Thank you. It works in a way of reloading. Since the feature enabled, page loading in general becomes somehow unpleasant and slow though. Quite dilemmatic whether it enables or disables.

**Comment by gorhill** [comment]
Use Firefox for Android, it does not suffer this issue. There is nothing else we can do, Chromium-based browser are lacking the proper framework to properlt block at browser launch. Another solution is to use uBO Lite.

**Comment by gorhill** [comment]
Right. it's written _desktop_ in title, I don't know why I read this as an Android issue.

**Comment by iam-py-test** [comment]
It's fine, mistakes happen
